test/parallel/test-tls-cipher-list.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+var common = require('../common');
```

Comment:
Well I tried #2306. Let's hear from @bnoordhuis. I have seen him driving for this in other PRs.